cpp/log/cert.cc

```diff
@@ -323,20 +323,22 @@ StatusOr<bool> Cert::HasBasicConstraintCATrue() const {
     return util::Status(Code::FAILED_PRECONDITION, "Cert not loaded");
   }
 
-  void* ext_struct;
-  Status status = ExtensionStructure(NID_basic_constraints, &ext_struct);
+  const StatusOr<void*> status = ExtensionStructure(NID_basic_constraints);
```

Comment:
or maybe `ext_struct`?
(naming `StatusOr` instances for the `Or` thing rather than the `Status` helps the readability I think)